src/allmydata/stats.py

```diff
@@ -1,54 +1,38 @@
 """
 Ported to Python 3.
 """
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals
-
-from future.utils import PY2
-if PY2:
-    from future.builtins import filter, map, zip, ascii, chr, hex, input, next, oct, open, pow, round, super, bytes, dict, list, object, range, str, max, min  # noqa: F401
-    from time import clock as process_time
-else:
-    from time import process_time
+
+from collections import deque
+from time import process_time
 import time
 
 from twisted.application import service
 from twisted.application.internet import TimerService
 from zope.interface import implementer
-from foolscap.api import eventually
 
 from allmydata.util import log, dictutil
 from allmydata.interfaces import IStatsProducer
 
 @implementer(IStatsProducer)
 class CPUUsageMonitor(service.MultiService):
-    HISTORY_LENGTH = 15
-    POLL_INTERVAL = 60  # type: float
+    HISTORY_LENGTH: int = 15
+    POLL_INTERVAL: float = 60
```

Comment:
It's not a float?

Reply:
I *guess* the way this works (but feel free to correct me) is that int is considered a subtype of float for the purposes of type checking so it is allowed and correct to assign both float and int values to a variable declared as of type float.

https://peps.python.org/pep-0483/#covariance-and-contravariance is the most directly relevant documentation I was able to easily dig up on the matter.